src/index.ts

```diff
@@ -8,9 +8,8 @@ export * from "generated/panda/css";
 export * from "generated/panda/jsx";
 export type { JsxStyleProps } from "generated/panda/types";
 
-// export backfill of react hot toast functions and types
-export { default as toast } from "react-hot-toast";
-export type { ToastPosition } from "react-hot-toast";
+// export backfill of sonner functions
+export { toast } from "sonner";
```

Comment:
Maybe export `ToastT` (potentially renaming beause that's not a great name) here as well?

Reply:
Actually, I guess we're exporting implementation types. So nevermind about this my thoughts have changed!